src/index.ts

```diff
@@ -413,13 +396,15 @@ export class GalleryComponent extends BaseComponent {
       thumb.initialHeight = medianHeight;
     }
 
-    this._$thumbs.link($.templates.galleryThumbsTemplate, this._thumbs);
+    const renderedHtml = this._thumbs.map(this._galleryThumbsTemplate).join('');

```

Comment:
create all the subset thumbnail html and then join them together to build up the full html.